appveyor.yml

```diff
@@ -64,28 +70,39 @@ environment:
       PYTHON: "3.6"
       ARCH: "64"
       CONFIGURATION: "Release"
+      BUILD_SCRIPT: "CMake_Build_Script"
     - JOB: "Build"
       GENERATOR: NMake Makefiles
       PYTHON: "3.6"
       ARCH: "64"
       CONFIGURATION: "Release"
+    - JOB: "Rust_Stable"
+      RUST_VERSION: stable
+      TARGET: x86_64-pc-windows-msvc
+      BUILD_SCRIPT: "Rust_Stable"
+      USE_CLCACHE: false
 
   MSVC_DEFAULT_OPTIONS: ON
   BOOST_ROOT: C:\Libraries\boost_1_63_0
   BOOST_LIBRARYDIR: C:\Libraries\boost_1_63_0\lib64-msvc-14.0
   APPVEYOR_SAVE_CACHE_ON_ERROR: true
-  USE_CLCACHE: true
 
 install:
-  - set MINICONDA=C:\Miniconda36-x64
-  - set PATH=%MINICONDA%;%MINICONDA%/Scripts;%MINICONDA%/Library/bin;%PATH%
-  - call ci\appveyor-setup.bat
+  - if NOT "%JOB%"=="Rust_Stable" set MINICONDA=C:\Miniconda36-x64
```

Comment:
Do you know if it's possible to have multi-line IF blocks instead?

Reply:
See above